### ISSUE TITLE
next branch broken because of caniuse-api

### ISSUE THREAD

**Issue by MoOx** [MoOx/statinamic#215]
https://travis-ci.org/MoOx/statinamic/jobs/112370214#L2058

I have write access to caniuse-api repo so can handle the fix if needed.

**Comment by MoOx** [comment]
See https://github.com/Nyalab/caniuse-api/issues/38

**Comment by thangngoc89** [comment]
Any luck on this ? Tests keep failing on Travis. 

For the first time, Appveyor tests don't fail. Haha

**Comment by MoOx** [comment]
No luck. Not related to caniuse-api directly (this package works great). 
I tried to reproduce locally with fresh install and same node/npm version, I don't have the issue. Will need to investigate on the error/stack trace...

**Comment by MoOx** [comment]
```
> caniuse-api@1.4.1 postinstall /home/travis/build/MoOx/statinamic/docs/node_modules/caniuse-api
> node generator.js
[BABEL] Note: The code generator has deoptimised the styling of "/home/travis/build/MoOx/statinamic/docs/node_modules/babylon/index.js" as it exceeds the max of "100KB".
/home/travis/build/MoOx/statinamic/docs/node_modules/babel-plugin-transform-react-jsx/lib/index.js:12
  var visitor = require("babel-helper-builder-react-jsx")({
                                                         ^
TypeError: require(...) is not a function
    at exports.default (index.js:12:17)
    at Function.memoisePluginContainer (/home/travis/build/MoOx/statinamic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:136:13)
    at Function.normalisePlugin (/home/travis/build/MoOx/statinamic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:170:32)
    at /home/travis/build/MoOx/statinamic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:210:30
    at Array.map (native)
    at Function.normalisePlugins (/home/travis/build/MoOx/statinamic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:182:20)
    at OptionManager.mergeOptions (/home/travis/build/MoOx/statinamic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:298:36)
    at /home/travis/build/MoOx/statinamic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:370:14
    at /home/travis/build/MoOx/statinamic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:390:24
    at Array.map (native)


...


npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.8
npm ERR! Linux 3.13.0-40-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v5.7.0/bin/node" "/home/travis/.nvm/versions/node/v5.7.0/bin/npm" "install"
npm ERR! node v5.7.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! caniuse-api@1.4.1 postinstall: `node generator.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the caniuse-api@1.4.1 postinstall script 'node generator.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the caniuse-api package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node generator.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs caniuse-api
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls caniuse-api
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/MoOx/statinamic/docs/npm-debug.log
```

**Comment by thangngoc89** [comment]
can't reproduce either. tried with docker

**Comment by MoOx** [comment]
Similar to https://github.com/reactuate/reactuate/issues/1.
It seems like an old issue, not sure why we do get this now...
- https://github.com/reactuate/reactuate/commit/dbb9ff08f8327a50fc552c8255ca5f2129ae3d59
- https://phabricator.babeljs.io/T2917
- https://phabricator.babeljs.io/T7053
- https://phabricator.babeljs.io/T6825
- http://stackoverflow.com/questions/35002195/babel-node-fails-to-require-jsx-file-in-node-modules

Hum maybe that's related to this change after all https://github.com/MoOx/statinamic/commit/2f86f35d3de515388504f54529031d51b0ac1715
I think moving nyc right before ava call (and not in the entire "test" script) should fix the thing. Will try tomorrow, time to sleep for me now.

**Comment by MoOx** [comment]
caniuse related issue has been fixed, now it's babel 6.6...
